### PR TITLE
Add shared latex files and rst color roles

### PR DESCRIPTION
**Pull request by gardner48**
* Add shared files for the latex preamble and cover page
* Add rst roles for text color
* Update user skeletons to gray out unchanged steps

### CODE REVIEW

File: doc/sundials_developers/source/conf.py [hunk 1]
```diff
@@ -11,238 +9,194 @@
...
+numfig_format = {
+  'section': '§%s'
 }
 
-
-# -- Options for HTML output -------------------------------------------------
+rst_prolog = open('../../shared/global.rst.txt', 'r').read()
+
+rst_epilog = """
+.. |YEAR| replace:: 2021
```
Comment (drreynolds) [inline doc/sundials_developers/source/conf.py:105]: Should this be 2022?

**Review by drreynolds — DISMISSED** [review]
I made a few changes to the recommended BibTeX entries for each package on the main superbuild page.  I also and added Rujeko to the ARKODE author list, and added Steven to the SUNDIALS contributor list (thanks to his "deduce" implicit RHS work).

I left one comment/question about the "year" -- I'm relatively certain this should be 2022 but wanted to verify before unilaterally changing.  Other than that item, I think this looks great.

File: doc/sundials_developers/source/conf.py (hunk 1 above)
Comment (gardner48) [reply doc/sundials_developers/source/conf.py:105]: Yes, Cody has addressed this in #152. The year will now get automatically updated by the version update script.

**Review by drreynolds — DISMISSED** [review]

File: doc/cvodes/guide/source/Usage/FSA.rst [hunk 2]
```diff
@@ -55,199 +55,186 @@ function to be called or macro to be referenced.
...
 
-      Typically, the quadrature variables should be initialized to 0.
+#. **Define the sensitivity problem**
 
-   #. **Define the sensitivity problem**
+   -  **Number of sensitivities** (*required*)
 
-      -  **Number of sensitivities** (*required*)
-         Set ``Ns`` :math:`= N_s`, the number of parameters with respect to which sensitivities are to be computed.
+      Set ``Ns`` :math:`= N_s`, the number of parameters with respect to which sensitivities are to be computed.
```
[the remark this answers is not in the document]
Comment (gardner48) [reply doc/cvodes/guide/source/Usage/FSA.rst:105]: It looks good to me (I did notice the text for the step before this one needed to be updated though).

**Review by balos1 — APPROVED** [review]

**Pull request merged**